sdk/eventhub/Azure.Messaging.EventHubs/tests/Primitives/EventProcessorTests.StartStop.cs

```diff
@@ -88,7 +88,7 @@ public async Task StartProcessingStartsTheProcessing(bool async)
             // Shut the processor down to ensure resource clean-up, but ignore any errors since it isn't the
             // subject of this test.
 
-            try { await mockProcessor.Object.StopProcessingAsync(cancellationSource.Token); }  catch {}
+            await mockProcessor.Object.StopProcessingAsync(cancellationSource.Token).IgnoreExceptions();
```

Comment:
This was a brief topic of conversation in a recent PR where it was noticed that we had multiple single-line try/catch blocks to keep tests focused by ignoring areas outside of the scope intended for the test.   This is an attempt to try and improve readability and declutter the formatting.

Open to thoughts on whether this approach works or does not for that goal.